app/waf/Dockerfile

```diff
@@ -0,0 +1,28 @@
+# build with:
+#   docker build . -t f5devcentral/f5-wafaas-openstack:latest
+
+# upload image with (authorized user only)
+#   docker push f5devcentral/f5-wafaas-openstack:latest
+
+FROM node:11.6.0
+
+LABEL maintainer="F5 Networks Inc. Corp."
+EXPOSE 3000/tcp
+
+WORKDIR /root/app_dir
+ADD dist dist
+RUN rm -r dist/test
+ADD public public
+COPY package.json index.js ./
```

Comment:
Maybe we can consider to use npm-shrinkwrap to reduce image size. Or, maybe we can consider it in the future.

Reply:
ok for it.